modules/mobile/app/controllers/mobile/v0/community_care_providers_controller.rb

```diff
@@ -52,7 +52,8 @@ def coordinates
       end
 
       def facility_coordinates
-        facility = Mobile::FacilitiesHelper.get_facilities(Array(params[:facilityId])).first
+        v1_facilities_flag = Flipper.enabled?(:mobile_v1_lighthouse_facilities, @current_user)
```

Comment:
not ideal but can't have the current user instance variable in the helpers module methods. This will be taken out once V0 has been sunset so no need to rework it for something temporary.